pkg/tfbridge/info.go

```diff
@@ -41,10 +47,14 @@ type ProviderInfo struct {
 	Python            *PythonInfo                // optional overlay information for augmented Python code-generation.
 	Golang            *GolangInfo                // optional overlay information for augmented Golang code-generation.
 	TFProviderVersion string                     // the version of the TF provider on which this was based
+	TFProviderLicense *TFProviderLicense         // license that the TF provider is distributed under. Default `MPL 2.0`.
 
 	PreConfigureCallback PreConfigureCallback // a provider-specific callback to invoke prior to TF Configure
 }
 
+// TFProviderLicense is a way to be able to pass a license type for the downstream Terraform provider.
```

Comment:
Isn't the TF provider an upstream relative to the bridged Pulumi provider?

Reply:
You are correct! Fixed